libs/designsystem/src/lib/components/slide-button/slide-button.component.scss

```diff
@@ -31,6 +31,7 @@
     display: flex;
     justify-content: center;
     align-items: center;
+    color: $slide-button-text-color;
```

Comment:
Why move this here?

Reply:
A rule of thumb is to never declare `background-color` without declaring `color` also (and vice versa) - and to always apply them to the same element/target (preferably within the same ruleset). The `color` property inherits its value when not declared explicitly, so `.slide-button-text` will inherit from here.

Reply:
Makes perfect sense, thanks :)